server/application/rest/work_shift.py

```diff
@@ -31,17 +31,29 @@
 ]
 
 
-@blueprint.route("/shifts", methods=["GET"])
+@blueprint.route("/shifts", methods=["GET", "POST"])
 @cross_origin()
-def list_work_sifts():
+def work_shifts():
     """
-    The function returns a list of all work shifts in the system.
+        On GET: The function returns a list of all work shifts in the system.
+        On POST: The function adds shifts to the system.
     """
-    repo = MemRepo(shifts)
-    result = workshift_list_use_case(repo)
-
-    return Response(
-        json.dumps(result, cls=WorkShiftJsonEncoder),
-        mimetype="application/json",
-        status=200,
-    )
+    if request.method == "GET":
+        repo = MemRepo(shifts)
+        result = workshift_list_use_case(repo)
+
+        return Response(
+            json.dumps(result, cls=WorkShiftJsonEncoder),
+            mimetype="application/json",
+            status=200,
+        )
+    elif request.method == "POST":
+        data = request.get_json()
+        shifts.extend(data)
```

Comment:
Instead of directly extending the data that we hard coded here for development purposes, you need to add a new use case for adding/creating a workshift. The usecase will be passed the repo object, and will call on the repo object to add the new workshift. That way, when we switch from using MemRepo to a real database, we will only need to instantiate a real database instead of MemRepo and the rest of the code will continue to work without changes.